### ISSUE TITLE
Ocaml Language Server not starting

### ISSUE THREAD

**Issue by Akin909** [onivim/oni#1377]
This issue is more likely than not an installation error but I've been fooling around a little with `Reason` and haven't been able to get the LSP to work in oni.

I've followed the install instructions from the `reason-react` website and it works in development but not in the packaged app I just get an error that says unable to spawn the lsp.

<img width="828" alt="screen shot 2018-01-26 at 23 26 21" src="https://user-images.githubusercontent.com/22454918/35465106-ac65c3c4-02f1-11e8-9b13-335145f4868a.png">

So far I haven't looked much at Oni LSP implementation so not sure if I've missed a configuration option or there might be an error somewhere.

I'm using the default config options (I haven't added any of my own)

**Comment by bryphe** [comment]
Oh wow - looks like we do have the wrong path for the language server in the packed version! Nice catch, @Akin909 .

The `.bin` folder doesn't make it into the packaged build, but we have this in our default config:
```
export const ocamllanguageserverpath = platform.iswindows()
    ? path.join(__dirname, "node_modules", ".bin", "ocaml-language-server.cmd")
    : path.join(__dirname, "node_modules", ".bin", "ocaml-language-server")
```

Might be part of the reason why the test to verify a Reason file is failing in #1189 🤔 Thanks for logging this issue!

**Comment by Akin909** [comment]
closed by #1380
